host-interaction/network/connectivity/set-tcp-connection-state.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: set TCP connection state
+    namespace: host-interaction/network/connectivity
+    authors:
+      - "@johnk3r"
+    description: The SetTcpEntry function sets the state of a TCP connection.
+    scope: function
+    att&ck:
+      - Defense Evasion::Impair Defenses [T1562]
```

Comment:
Can we be sure this is always the technique here?

Reply:
MSFT describes the following:

1 - **The SetTcpEntry function sets the state of a TCP connection.**
2 - **Currently, the only state to which a TCP connection can be set is MIB_TCP_STATE_DELETE_TCB.**

So in my understanding yes. Do you have any other experience with this?

Reply:
I see, should we then add `number: 12 = MIB_TCP_STATE_DELETE_TCB` to the logic?

Reply:
Nice ideia!


![image](https://github.com/mandiant/capa-rules/assets/6247648/8774aaf0-f187-4346-a07a-57d7b31283aa)

What do you think?

Reply:
looks good, please add it to the rule and then we're g2g